pywhat/what.py

```diff
@@ -53,7 +53,7 @@ def parse_options(rarity, include_tags, exclude_tags):
 )
 @click.argument("text_input", required=True)
 @click.option("-t", "--tags", is_flag=True, expose_value=False, callback=print_tags, help="Show available tags and exit.")
-@click.option("-r", "--rarity", help="Filter by rarity. This is in the range of 0:1. To filter only items past 0.5, use 0.5: with the colon on the end.")
+@click.option("-r", "--rarity", help="Filter by rarity. Rarity is how unlikely something is to be a false-positive. The higher the number, the more unlikely. This is in the range of 0:1. To filter only items past 0.5, use 0.5: with the colon on the end. Default 0.1:1", default="0.1:1")

```

Comment:
My hope with `default="0.1:1"` was that it'd display in the help menu. Sadly it does not, but I think it should and it might in the future? or it might currently and i broke my local click install? :shrug: